tensorflow/docs_src/programmers_guide/graphs.md

```diff
@@ -319,7 +319,7 @@ described below.
 * **`target`.** If this argument is left empty (the default), the session will
   only use devices in the local machine. However, you may also specify a
   `grpc://` URL to specify the address of a TensorFlow server, which gives the
-  session access to all devices on machines that that server controls. See
```

Comment:
I'm not sure if that is a typo. :)

the second that is a determiner for "server". Consider changing it to 'the" .